Carbon.Cryptography/Functions/Get-CCertificate.ps1

```diff
@@ -37,6 +37,13 @@ function Get-CCertificate
     When loading certificates from a certificate store, `Get-CCertificate` adds `StoreLocation` and `StoreName`
     properties for the store where the certificate was found.
 
+    When loading a certificate from a file and that certificate contains a private key, Windows will temporarily write

```

Comment:
I really think this should be the default? I think most users would expect this behavior and would be surprised to find out just loading a cert creates a file in the cert stores.

Reply:
If we have to, we can release a new major version. It's a breaking change.

Reply:
That was my original intention, to make `EphemeralKeySet` the default. But then I found that when using `EphemeralKeySet` on Windows PowerShell the `.PrivateKey` property on the certificate object is null (not so on PowerShell Core). Due to that and the fact that .NET itself also doesn't default to `EphemeralKeySet`, I left the default as `UserKeySet` (same as .NET).

However, after taking another look I just found that when using `EphemeralKeySet` on Windows PowerShell you can use [RSACertificateExtensions.GetRSAPrivateKey(X509Certificate2)](https://learn.microsoft.com/en-us/dotnet/api/system.security.cryptography.x509certificates.rsacertificateextensions.getrsaprivatekey?view=netframework-4.6.2) to get the private key object instead of the `.PrivateKey` property (which is null).

This would be a breaking change and we'd need to update the docs to let users know that they need to use `.GetRSAPrivateKey()` to access the private key on a certificate returned from `Get-CCertificate`, instead of `.PrivateKey`.

The change in this PR should still be merged as it prevents private keys from being left on disk when just reading PFX files.

If you're in agreement, I can follow up with a separate PR that implements the `EphemeralKeySet` and `.GetRSAPrivateKey()` changes in a new major version.

Reply:
If the .NET default is `UserKeySet` , let's leave that as our default to. I don't think any more work is required.